<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>281</width>
      <height>190</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Deals">
     <attribute name="title">
      <string>Сделка</string>
     </attribute>
     <widget class="QLineEdit" name="product_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Товар</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="customer_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Покупатель</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата сделки</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="product_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="customer_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="kolvo_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Количество</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QCheckBox" name="isopt_input">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>130</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Признак опта</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="kolvo_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Customers">
     <attribute name="title">
      <string>Покупатели</string>
     </attribute>
     <widget class="QLineEdit" name="name_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Контактное лицо</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Products">
     <attribute name="title">
      <string>Товары</string>
     </attribute>
     <widget class="QLineEdit" name="title_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="title_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Наименование</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="optprise_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Оптовая цена</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="rozprise_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Розничная цена</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="descr_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="descr_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Описание</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="optprise_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="rozprise_input">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="input">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
